test/FunctionalTests/Microsoft.OData.Core.Tests/ScenarioTests/UriBuilder/PathBuilderTests.cs

```diff
@@ -126,6 +126,30 @@ public void BuildPathWithEscapeMarkerWithTypeSegmentInKeyAsSegment()
             Assert.Equal(new Uri("http://gobbledygook/People/Fully.Qualified.Namespace.Employee"), actualUri);
         }
 
+        [Fact]
+        public void BuildPathWithEachSegmentRelativeUri()
+        {
+            Uri queryUri = new Uri("People/$each/Fully.Qualified.Namespace.SummonPuppies", UriKind.Relative);
+
+            Uri actualUri = UriBuilder(queryUri, ODataUrlKeyDelimiter.Parentheses, settings);
+            Assert.Equal(new Uri("http://gobbledygook/People/$each/Fully.Qualified.Namespace.SummonPuppies"), actualUri);
+
+            actualUri = UriBuilder(queryUri, ODataUrlKeyDelimiter.Slash, settings);
+            Assert.Equal(new Uri("http://gobbledygook/People/$each/Fully.Qualified.Namespace.SummonPuppies"), actualUri);

```

Comment:
>new Uri("http://gobbledygook/People/$each/Fully.Qualified.Namespace.SummonPuppies" [](start = 25, length = 82)

just define this once and use it in both asserts

Reply:
:+1: